<?xml version='1.0'?>
<launch>
	
	<node pkg="ml_detector" type="transform_test.py" name="transform_test" output="screen">
	</node>

	
</launch>
